app/src/main/java/com/example/internetshop/presentation/activity/MainActivity.kt

```diff
@@ -89,24 +92,24 @@ class MainActivity : AppCompatActivity(), ContainerHolder {
     private fun setupAppBar(navController: NavController) {
         WindowCompat.setDecorFitsSystemWindows(window, false)
         setSupportActionBar(binding?.toolbar)
+        val topLevelDestinations = setOf(
```

Comment:
Define this set only in toolbar view model. And in app bar config use `toolbarViewModel.topLevelDestinations`

Reply:
fixed